ray-on-gke/tpu/kuberay-tpu-webhook/main.go

```diff
@@ -635,6 +622,34 @@ func init() {
 func main() {
 	flag.Parse()
 
+	// use in-cluster config if kubeConfig path is not passed as a flag
+	var client *kubernetes.Clientset
+	if KubeConfigPath == "" {
+		config, err := rest.InClusterConfig()
+		if err != nil {
+			panic(err)
+		}
+		client = kubernetes.NewForConfigOrDie(config)
+	} else {
+		config, err := clientcmd.BuildConfigFromFlags("", KubeConfigPath)
+		if err != nil {
+			panic(err)
+		}
+		client = kubernetes.NewForConfigOrDie(config)
+	}
+
+	// instantiate PodInformer for Ray worker pods in the GKE cluster
+	tweakListOptionsFunc := func(options *metav1.ListOptions) {
+		options.LabelSelector = "ray.io/node-type=worker,app.kubernetes.io/created-by=kuberay-operator"
+	}
+	factory := informers.NewFilteredSharedInformerFactory(client, 5*time.Minute, metav1.NamespaceAll, tweakListOptionsFunc)
+	podLister = factory.Core().V1().Pods().Lister()
+
+	// start the PodInformer and wait for cache sync
+	stopCh := make(chan struct{})
```

Comment:
Close the stopCh channel on termination, which will signal the informer to stop as well

Reply:
Done in [f038d21](https://github.com/GoogleCloudPlatform/ai-on-gke/pull/723/commits/f038d21bda9d7f8bd7df14de6e401200ab2dae4a).